languages/java8/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,54 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+class Algorithm {
+    public String apply(String input) throws RuntimeException {
+        return "hello ".concat(input);
+    }
+
+    public static void main(String[] args) {
+        Algorithm defs = new Algorithm();
+        Handler algo = new Handler<>(defs.getClass(), defs::apply);
+        algo.serve();
+    }
+}
+
+
+/**
+ * This class below describes a more advanced template.
+ * If your algorithm needs key data or model files at runtime, you don't want to be downloading that every time your algorithm gets called.
+ * In this situation, you will want to designate a `Loading` function (like DownloadModel below), and provide that method to the AlgorithmHandler constructor.
+ * For more information, please refer to the advanced user guide in https://docs.algorithmia.com
+ */
+
+//public class AlgorithmAdvanced {
```

Comment:
Just want to make sure, even though this is an Algorithm.java class, the name of the class holding main, can me anything?

Reply:
Yes except they'll need to change the main class in pom.xml as well, otherwise it won't package properly